smart_campus/app/models.py

```diff
@@ -227,12 +228,20 @@ def __str__(self):
 
 class StationImage(models.Model):
     station = models.ForeignKey('Station', on_delete=models.CASCADE)
-    image = models.ImageField(upload_to='app/images/station/')
+    image = models.ImageField(upload_to='media/images/station/')
     is_primary = models.BooleanField(default=False)
 
     def __repr__(self):
         return 'Image {img_id}'.format(img_id=self.id)
 
+    def save(self, *args, **kwargs):
+        super(StationImage, self).save(*args, **kwargs)
+
+        # auto save the url of the primary image to the station
+        if self.is_primary == True:
```

Comment:
Typically, we do not use `==` to compare with `True` and `False`.
`self.is_primary is True` is more pythonic.
Is there any reason that you don't do so?

Reply:
Uh no, I'll  fix this